src/components/Modal/index.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./styles.module.css";
+import { ImCross } from "react-icons/im";
+
+export const Modal = ({ modalRef, openModal, handleModalClose, children }) => {
+  if (!openModal) {
+    return <></>;
+  }
+
+  return (
+    <>
+      <div id={styles.backdrop}>.</div>
+      <div className={styles.container}>
+        <div className={styles.modal} ref={modalRef}>
+          <button className={styles.closeBtn} onClick={handleModalClose}>
```

Comment:
P2:
button type 명시해주세요.

```suggestion
          <button type='button' className={styles.closeBtn} onClick={handleModalClose}>
```